constellations/gemini-admin/src/user/userEdit.js

```diff
@@ -0,0 +1,379 @@
+                                    message: "You must input an email",
+                                },
+                                {
+                                    validate: (email) =>
+                                        /^[a-zA-Z0-9]+@[a-zA-Z]+\.[a-zA-Z]+/.test(
+                                            email
+                                        ),
+                                    message: "Invalid email",
+                                },
+                            ]}
+                        />
+
+                        <InputText
+                            label="Phone"
+                            value={this.state.phone}
+                            onChangeCallback={(e) =>
+                                this.handleChange(e, "phone")
+                            }
+                            description="Enter your phone number"
+                        />
+
+                        <h2 id="guardian">Is this user a guardian?</h2>
+                        <input
+                            type="checkbox"
+                            checked={this.state.isGuardian}
+                            onChange={this.onChangeGuardian}
+                        />
+                        <span>Yes</span>
+
+                        {/*
+                        It doesn't look like there's anything in orion for these, so I left
+                        them out for now
+                        <InputText
+                            label="School"
+                            value={this.state.school}
+                            onChangeCallback={(e) => this.handleChange(e, "school")}
+                            required={false}
+                            description="Enter your school"
+                        />
+                        <InputText
+                            label="Graduation Year"
+                            value={this.state.gradYear}
+                            onChangeCallback={(e) => this.handleChange(e, "gradYear")}
+                            required={true}
+                            description="Enter your graduation year"
+                        />*/}
+                    </div>
+                    {associatedAccount}
+                </div>
+
+                <InputText
+                    label="Notes"
+                    isTextBox={true}
+                    value={this.state.notes}
+                    onChangeCallback={(e) => this.handleChange(e, "notes")}
+                    description="Add any notes"
+                />
+
+                <div className="buttons">
+                    <button onClick={this.onClickCancel} className="btn-cancel">
+                        Cancel
+                    </button>
+                    {deleteButton}
+                    <div className="spacer"></div>
+                    <button onClick={this.onClickSave} className="btn-save">
+                        Save
+                    </button>
```

Comment:
Interesting use of the "spacer" div.

You could try something like this: (This is shorthand)
```
<div buttons>
    <div buttons-left>
        <button cancel>
        <button delete>
    </div>
    <div buttons-right>
        <button save>
    </div>
</div>
```

That way, the left buttons are bundled together and the right buttons (i know there's one - it's possible you don't need the wrapping div "buttons-right") are bundled together.
Then you can use `justify-content: space-between` to force the left and right buttons to the ends of the flex-container.